### PR TITLE
Added some juicy gifs

### PR DESCRIPTION
**Pull request by brianhdev**
# Proposed gifs to be added:
Pacha: 

Terminator:

Ship in Rough Water

I Ship Them So Hard

I Ship It!

Let’s Set Sail

### CODE REVIEW

**Comment by brianhdev** [comment]
@seantomburke some juicy gifs ready for review

**Review by seantomburke — APPROVED** [review]

**Comment by seantomburke** [comment]
https://giphy.com/gifs/loop-ship-seas-bZBmitwUwKtDa

File: gifs.json [hunk 1]
```diff
@@ -419,6 +419,48 @@
...
+      "_id": 64,
+      "name": "ooishipit",
+      "url": "https://media.giphy.com/media/4ZgwMNUYMxHIyFXujF/giphy.gif",
+      "description": "Oooo I ship it!!!",
+      "active": 1
+    },
+    {
+      "_id": 65,
+      "name": "setsail",
+      "url": "https://i.giphy.com/media/oGETnrfibydbDKKysk/giphy.webp",
```
Comment (seantomburke) [inline gifs.json:461]:
```suggestion
      "url": "https://media.giphy.com/media/oGETnrfibydbDKKysk/giphy.gif",
```

File: gifs.json [hunk 2]
```diff
@@ -419,6 +419,48 @@
...
+      "_id": 61,
+      "name": "terminator",
+      "url": "https://media.giphy.com/media/gFwZfXIqD0eNW/giphy.gif",
+      "description": "I know now why you cry, but it's something I can never do...",
+      "active": 1
+    },
+    {
+      "_id": 62,
+      "name": "roughwater",
+      "url": "https://i.giphy.com/media/bZBmitwUwKtDa/giphy.webp",
```
Comment (seantomburke) [inline gifs.json:440]:
```suggestion
      "url": "https://media.giphy.com/media/bZBmitwUwKtDa/giphy.gif",
```

**Pull request merged**